src/main/java/decodes/cwms/CwmsSqlDatabaseIO.java

```diff
@@ -323,97 +337,34 @@ public IntervalDAI makeIntervalDAO()
 	public Connection getConnection()
 	{
 		// Called from DAOs and DbIo to get a new connection from the pool.
-		if (conInfo == null || conInfo.getLoginInfo() == null)
+		if (conInfo == null || conInfo.getLoginInfo() == null || pool == null)
 		{
 			Logger.instance().failure(module + ".getConnection -- loginInfo is null! DB not initialized?");
 			return null;
 		}
-		
-		Connection ret = null;
 		try
 		{
-			ret = CwmsDbConnectionPool.getInstance().getConnection(conInfo.getLoginInfo(), module);
+			return pool.getConnection();
 		}
-		catch (SQLException ex)
-		{
-			String msg = module + ".getConnection() Cannot get CWMS connection for user '"
-				+ conInfo.getLoginInfo().getUser() + "' and dbURI '" 
-				+ conInfo.getLoginInfo().getUrl() + "'";
-			Logger.instance().failure(msg);
-			PrintStream ps = Logger.instance().getLogOutput();
-			if (ps != null)
-				ex.printStackTrace(ps);
-			else
-				ex.printStackTrace(System.err);
-			return null;
-			// TODO Should this method throw BadConnectException?
-		}
-
-		// Force autoCommit on.
-		try{ ret.setAutoCommit(true); }
 		catch(SQLException ex)
 		{
-			Logger.instance().warning(module + ".getConnection() Cannot set SQL AutoCommit to true: " + ex);
-		}
-
-		try
-		{
-			CwmsTimeSeriesDb.setCtxDbOfficeId(ret, dbOfficeId, conInfo.getDbOfficeCode(), conInfo.getDbOfficePrivilege());
-		}
-		catch (Exception ex)
-		{
-			String msg = module + ".getConnection() Cannot set VPD context username/officeId username='" 
-				+ conInfo.getLoginInfo().getUser()
-				+ "', officeId='" + dbOfficeId + "', officeCode=" + conInfo.getDbOfficeCode() + ", priv='"
-				+ conInfo.getDbOfficePrivilege() + "':" + ex;
-			Logger.instance().failure(msg);
-			try { CwmsDbConnectionPool.close(ret); } catch(Exception ex2) {}
-			return null;
-			// TODO Should this method throw BadConnectException?
-//			throw new BadConnectException(msg);
+			Logger.instance().fatal("Unabled to get connection " + ex.getLocalizedMessage());
```

Comment:
typo `Unable`, also do we really need a log line here when we are throwing the RuntimeException?